README.md

```diff
@@ -162,6 +164,9 @@ The following table summarises all the available resource URIs, and the effect o
 | [/readers/{id}/authorisedDevices](#authorised-device) | N/A                                                 | N/A                                   | N/A                               | Removes all authorised devices for a reader |
 | [/readers/{id}/authentication](#authentication)       | N/A                                                 | N/A                                   | Authenticates a reader's password | N/A                                         |
 | [/targetedNotifications](#targeted-notifications)     | N/A                                                 | Sends a targeted notification         | N/A                               | N/A                                         |
+| [/{path:editions|publications}/{nodeId}/storedFiles](#stored-file)     | Returns a list of stored files associated with a node | N/A          | N/A                               | N/A                                         |
+| [/{path:editions|publications}/{nodeId}/storedFiles/{id}](#stored-file)     | Returns the details of a single stored file associated with a node | N/A          | N/A                               | N/A                                         |
+| [/{path:editions|publications}/{nodeId}/storedFiles/content/{id}](#stored-file)     | Returns base 64 byte array version of the content of a stored file | N/A          | N/A                               | N/A                                         |
```

Comment:
`{nodeId}/storedFiles/{id}/content` would make more sense as the content resource belongs to the `storedFile`. Your explanation of base 64 encoding could also do with some clarification. I would imagine providing the file as binary data with the correct MIME type would be more intuitive.